cijoe/tests/apps/test_fio.py

```diff
@@ -76,3 +76,77 @@ def test_fio_engine_iov(cijoe, device, be_opts, cli_args):
         env=env,
     )
     assert not err
+
+
+@xnvme_parametrize(labels=["zns", "bdev"], opts=["be", "admin", "sync", "async", "mem"])
+def test_fio_engine_zns(cijoe, device, be_opts, cli_args):
+    """
+    The construction of the fio-invocation is done in 'cijoe.fio.wrapper.fio_fancy'
+    """
+    if be_opts["be"] == "fbsd":
+        pytest.skip(reason="Freebsd kernel doesn't support zns")
+    if be_opts["be"] == "linux" and be_opts["sync"] in ["psync", "block"]:
+        pytest.skip(reason="[sync=psync,block] does not support mgmt. send/receive")
+    if be_opts["be"] == "linux" and be_opts["async"] in ["thrpool"]:
+        pytest.skip(reason="[async=thrpool] gives Zone Invalid Write")
+
+    is_windows = be_opts["mem"] == "windows" or be_opts["be"] == "windows"
+    fio_output_fpath = Path("C:/tmp" if is_windows else "/tmp") / "fio-output.txt"
+
+    size = "64M"
+    # size = "1G"
+
+    env = {}
+
+    hugepage_path = cijoe.config.options.get("hugetlbfs", {}).get("mount_point", "")
```

Comment:
You can use `cijoe.getconf("hugetlbfs.mount_point", "")` (added in v0.9.38 of cijoe iirc)

Reply:
Done